src/cr/cube/dimension.py

```diff
@@ -873,13 +873,13 @@ def element_ids(self):
 
     @lazyproperty
     def insertion_id(self):
-        """int insertion-id in the "insertion_id" field of the transform dict.
+        """int insertion-id in the "id" field of the transform dict.
 
-        Raises KeyError if this transform dict does not contain an "insertion_id" field.
-        Note that not all order types use the "insertion_id": field but this field is
-        required in all that do.
+        Raises KeyError if this transform dict does not contain an "id" field. Note that
+        not all order types use the "id": field but this field is required in all that
+        do.
```

Comment:
The `Note that not all ...` sentence seems like a cut-and-paste error from when I created that in the sort-by-value PR. I'm thinking that doesn't really fit in this context, do you? Maybe we should cut that sentence.